### PR TITLE
Added standardjs and multitab logout

### PR DESCRIPTION
**Pull request by impronunciable**

### CODE REVIEW

**Comment by jaredpalmer** [comment]
Try actually using `getInitialProps` in one of the protected pages...it is not getting passed thru via the hoc.

`Page.getInitialProps is not function`

**Comment by impronunciable** [comment]
@jaredpalmer sorry I don't understand the issue

File: hocs/defaultPage.js [hunk 1]
```diff
@@ -1,43 +1,64 @@
...
+  })
+}
 
 export default Page => class DefaultPage extends React.Component {
-  static getInitialProps(ctx) {
-    const loggedUser = process.browser ? getUserFromLocalStorage() : getUserFromCookie(ctx.req);
-    const pageProps = Page.getInitialProps && Page.getInitialProps(ctx);
+  static getInitialProps (ctx) {
+    const loggedUser = process.browser ? getUserFromLocalStorage() : getUserFromCookie(ctx.req)
+    const pageProps = Page.getInitialProps && Page.getInitialProps(ctx)
```
Comment (jaredpalmer) [inline hocs/defaultPage.js]: `pageProps` is always undefined.

File: pages/secret.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
     fontSize: 20,
     fontWeight: 200,
-    lineHeight: '30px',
-  }),
-};
+    lineHeight: '30px'
+  })
+}
 
 const Secret = ({ loggedUser }) => (
```
Comment (jaredpalmer) [inline pages/secret.js]: Try a making securePage that uses getInitialProps...it won't work.

Comment (luisrudge) [inline pages/secret.js]: ah, yeah.. I need to relay the `getInitialProps` from both HOCs. will do that shortly. thanks for catching that

**Comment by luisrudge** [comment]
thanks for everything @impronunciable. That's awesome. I didn't know you could add a listener for localStorage changes. That's super cool!

File: pages/secret.js (hunk 2 above)
Comment (luisrudge) [inline pages/secret.js]: @jaredpalmer I just fixed this [here](https://github.com/luisrudge/next.js-auth0/commit/d2b355fa64c3ded95492b36041627d4eb5bedb7d)

**Comment by luisrudge** [comment]
@impronunciable this version is live on https://nextjs-auth0.now.sh/

Thanks again!

**Comment by luisrudge** [comment]
@impronunciable we have an issue.. the `now` command doesn't send gitignored files, so it's not sending `config.json` (:

**Comment by sedubois** [comment]
@luisrudge did you fix that issue of `config.json` not deploying? And it's also hard to pass environment variables as discussed here: https://github.com/zeit/next.js/issues/159

**Comment by luisrudge** [comment]
@sedubois no, I didn't.. I just published an alternative version on `now` using a `config2.json` file. :trollface:

**Comment by sedubois** [comment]
Hah I hope we can do better 😄

**Pull request merged**